docs/book/src/SUMMARY.md

```diff
@@ -31,6 +31,7 @@
   - [Rapid iterative development with Tilt](./developer/tilt.md)
   - [Guide for API conversions](./developer/conversion.md)
   - [Release Process](./developer/release.md)
+  - [Contributing Guidelines](./developer/contributing-guidelines.md)
```

Comment:
```suggestion
  - [Release Support Guidelines](./developer/release-support-guidelines.md)
```

@Niharika0306 Lets rename this as above.

Reply:
Yes, makes sense. I have addressed it.